docs/content/preview/architecture/docdb/persistence.md

```diff
@@ -90,7 +90,7 @@ A binary-comparable encoding is used for translating the value for each YCQL typ
 
 ## Packed row format
 
-Packed row format support is currently in [Early Access](/preview/releases/versioning/#feature-availability).
+Packed row format for the YSQL API is [Generally Available](/preview/releases/versioning/#feature-availability) as of 2.20.0 release, while the support for Packed Row feature in the YCQL API is still in Tech Preview (/preview/releases/versioning/#feature-availability).
```

Comment:
@ddhodge - do we prefer GA or Generally Available here?

Reply:
Dwight responded.